manifest.webapp

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.1",
+  "version": "0.2",
```

Comment:
I'd recommend switching to a version system like Bower uses, i.e. ```x.y.z```. This looks like a big version jump, but it's not.